applications/StructuralMechanicsApplication/custom_elements/membrane_element.cpp

```diff
@@ -1154,12 +1155,10 @@ void MembraneElement::AddExplicitContribution(
             array_1d<double, 3>& r_nodal_inertia = GetGeometry()[i].GetValue(NODAL_INERTIA);
             SizeType index = i * dimension;
 
-            #pragma omp atomic
-            r_nodal_mass += mass_vector[index];
+            AtomicAdd(r_nodal_mass, mass_vector[index]);
 
             for (SizeType k = 0; k < dimension; ++k) {
-                #pragma omp atomic
-                r_nodal_inertia[k] += 0.0;
+                AtomicAdd(r_nodal_inertia[k], 0.0);
```

Comment:
What is the use of this line, where we are adding to zero to each element in the array? Or is it just a standard formula where the constant which they are adding is zero here?

Reply:
@KlausBSautter